dcspy/logitech.py

```diff
@@ -13,15 +13,16 @@
 from dcspy.aircraft import BasicAircraft, MetaAircraft
 from dcspy.dcsbios import ProtocolParser
 from dcspy.models import SEND_ADDR, SUPPORTED_CRAFTS, Gkey, KeyboardModel, LcdButton, LcdColor, LcdMono, ModelG13, ModelG15v1, ModelG15v2, ModelG19, ModelG510
-from dcspy.sdk import key_sdk, lcd_sdk
+from dcspy.sdk import lcd_sdk
+from dcspy.sdk.key_sdk import GkeySdkManager
 from dcspy.utils import get_full_bios_for_plane, get_planes_list
 
 LOG = getLogger(__name__)
 
 
 class KeyboardManager:
     """General keyboard with LCD from Logitech."""
-    def __init__(self, parser: ProtocolParser, **kwargs) -> None:
+    def __init__(self, parser: ProtocolParser, socket: socket, **kwargs) -> None:
```

Comment:
I needed to pass the DCS socket into the constructor so that it can be used in the callback handler. This is still being directly passed to the other function that sends requests that aren't gkey presses for push buttons so perhaps that code should be refactored as well vs. passing it in the `_handle_connection` function.